modules/currency-l1/src/test/scala/org/tessellation/currency/l1/http/DataApplicationRoutesSuite.scala

```diff
@@ -149,10 +145,10 @@ object DataApplicationRoutesSuite extends HttpSuite {
 
     forall(signedOf(dummyUpdateGen)) { update =>
       for {
-        dataQueue <- ViewableQueue.make[F, Signed[DataUpdate]]
-        _ <- dataQueue.offer(update)
+        dataQueue <- ViewableQueue.make[F, NonEmptyList[Signed[DataTransaction]]]
+        _ <- dataQueue.offer(NonEmptyList.of(update.asInstanceOf[Signed[DataTransaction]]))
```

Comment:
you can use `NonEmptyList.one`and you can pass a type there `NonEmptyList.one[A]`

Reply:
👍🏼